src/crepe/api/Text.cpp

```diff
@@ -0,0 +1,14 @@
+#include "../facade/FontFacade.h"
+
+#include "Text.h"
+
+using namespace crepe;
+
+Text::Text(game_object_id_t id, const vec2 & dimensions, const vec2 & offset,
+		   const std::string & font_family, const Data & data, const std::string & text,
+		   std::optional<Asset> font)
```

Comment:
Waarom kun je direct een font meegeven aan Text? De font family bepaalt toch wat er in Text::font komt te staan? Als je wil dat de gebruiker ook een direct pad aan kan geven met een `const Asset &` kun je beter een overloaded constructor voor Text maken maar dat is extra functionaliteit die niet hoeft van mij en beter als nice to have ticket in miro kan.